models/User.js

```diff
@@ -16,9 +16,14 @@ module.exports = (sequelize, DataTypes) => {
       allowNull: false
 
     },
+    isMentor: {
+      type: DataTypes.BOOLEAN,
+      allowNull: true,
+      defaultValue: false
+    },
   });
   User.associate = models => User.hasOne(models.Profile, {
-
+    foreignKey: 'userId'
```

Comment:
There should be a cascade command here so that if a user is deleted or the user table is deleted, corresponding profiles are obliterated.